bgpd/bgp_network.c

```diff
@@ -57,8 +57,26 @@ struct bgp_listener {
 	union sockunion su;
 	struct thread *thread;
 	struct bgp *bgp;
+	char *name;
 };
 
+void bgp_dump_listener_info(struct vty *vty)
+{
+	struct listnode *node;
+	struct bgp_listener *listener;
+
+	vty_out(vty, "Name             fd Address\n");
+	vty_out(vty, "---------------------------\n");
+	for (ALL_LIST_ELEMENTS_RO(bm->listen_sockets, node, listener)) {
+		char buf[128];
+
+		vty_out(vty, "%-16s %d %s\n",
```

Comment:
maybe this would be handy showing if it's a VRF or VIEW as well?

Reply:
This is listeners.  We have a listener per vrf effectively.  not per view so I am not sure what you are asking for?

Reply:
I just meant like:
```
default(vrf)   23 1.1.1.1
RS(view)       24 1.1.1.2
blue(vrf)      24 1.1.1.3
```
But maybe that's not relevant. Ignore then.

Reply:
I dont think this is relevant.